plugins/zenoh-plugin-rest/config_schema.json5

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Config",
+  "type": "object",
+  "required": [
+    "http_port"
+  ],
+  "properties": {
+    "__path__": {
+      "type": [
+        "string",
+        "null"
+      ]
+    },
+    "__required__": {
+      "type": [
+        "boolean",
+        "null"
+      ]
+    },
+    "http_port": {
+      "type": "string"
```

Comment:
The generated schema here is incomplete: a number would also be accepted. Not sure it that's an issue for our purpose though

Reply:
This goes directly from definition of port field in zenoh-plugin-rest/config.rs. If we want to correct it, we should fix this type
```
pub struct Config {
    #[serde(deserialize_with = "deserialize_http_port")]
    pub http_port: String,
    __path__: Option<String>,
    __required__: Option<bool>,
}
```